.storybook/preview.js

```diff
@@ -1,4 +1,4 @@
-import '../dist/app.css';
+import '@mattilsynet/designsystem-css/dist/app.css'
```

Comment:
Is it possible to remove `dist` from this. Ideally, I think it should be something like:
```suggestion
import '@mattilsynet/designsystem-css/app.css'
```